### PR TITLE
OGL: Use GL_EXT_texture_lod_bias on OpenGL ES if available

### PR DESCRIPTION
**Pull request by OatmealDome**
This adds support for [EXT_texture_lod_bias](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_lod_bias.txt) on OpenGL ES.

### CODE REVIEW

File: Source/Core/VideoCommon/PixelShaderGen.cpp [hunk 1]
```diff
@@ -648,14 +648,14 @@ uint WrapCoord(int coord, uint wrap, int size) {{
               "  float3 coords = float3(float(uv.x) / size_s, float(uv.y) / size_t, layer);\n");
     if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
     {
-      if (!host_config.backend_sampler_lod_bias)
+      /*if (!host_config.backend_sampler_lod_bias)
```
Comment (OatmealDome) [inline Source/Core/VideoCommon/PixelShaderGen.cpp:651]: The shader fallback path has been disabled for testing... Will drop this when ready

**Comment by JMC47** [comment]
It doesn't seem like any of our phones actually support the extension to test it lol.

**Comment by OatmealDome** [comment]
Was Jos's phone supported? I think they said their screen matched the screenshots taken on macOS, but maybe I misunderstood

**Comment by JosJuice** [comment]
My phone isn't supported. What I meant is that this PR matches the known broken rendering and master matches the known good rendering.

**Comment by OatmealDome** [comment]
Closing because of poor device support and minimal performance difference

**Pull request closed without merging**